src/nupic/regions/ScalarSensor.cpp

```diff
@@ -21,7 +21,7 @@
  */
 
 /** @file
- * Implementation of the ScalarSensor
+ * Implementation of the ScalarSensor Region
```

Comment:
Offtopic: but rename later to ScalarEncoderRegion? to be consistent

Reply:
> Offtopic: but rename later to ScalarEncoderRegion? to be consistent

yes, was thinking about doing that.
We also need to think about how we want to represent other encoders in the NetworkAPI.